lib/shopify_api/resources/fulfillment_order.rb

```diff
@@ -22,6 +22,11 @@ def self.find(scope, *args)
       end
     end
 
+    def self.release_hold(order_id:)
```

Comment:
We've extensively discussed why this action has to live on the `fulfillment order` resource and why we accept an order id here merely as a convenience method. If you need more details please let me know and I'll share the docs with you.